ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -473,7 +479,11 @@ def odf_overview_ui(
 
         if not all(res_dict.values()):
             failed_checks = [check for check, res in res_dict.items() if not res]
-            pytest.fail("Following checks failed: {}".format(failed_checks))
+            pytest.fail(
+                "Following checks failed. 1 - Pass, 0 - Fail. \n{}".format(
```

Comment:
It is a legend for the log message.

In case of failure it will look like this. I did not have patience  to replace "1" and "0" with readable True and False
so added a legend before printing this table

+------------+---------+
| check name | result      |
+=========+=======+
| check_a       | 1               |
+------------+---------+
| check_b       | 1              |
+------------+---------+
| check_3       | 0              |
+------------+---------+
| check_4       | 1              |
+------------+---------+
| check_5       | 1              |
+------------+---------+